.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateTimeParser.cs

```diff
@@ -34,6 +34,11 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refTime)
             if (er.Type.Equals(ParserName, StringComparison.Ordinal))
             {
                 var innerResult = MergeDateAndTime(er.Text, referenceTime);
+                if ((!innerResult.Success) && ((config.Options & DateTimeOptions.TasksMode) != 0))
```

Comment:
Add a blank line here as the two are unrelated.

Reply:
added a blank line and pushed the change.